kernel/kernel-api/src/test/scala/io/delta/kernel/internal/stats/FileSizeHistogramSuite.scala

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (2025) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.internal.stats
+
+import scala.collection.JavaConverters._
+import scala.collection.Searching.search
+
+import org.scalatest.funsuite.AnyFunSuite
+
+class FileSizeHistogramSuite extends AnyFunSuite {
+  implicit class FileSizeHistogramOps(histogram: FileSizeHistogram) {
+    def getBoundaries: List[Long] = histogram.sortedBinBoundaries.asScala.map(_.longValue()).toList
+    def getCounts: Array[Long] = histogram.fileCounts.map(_.longValue)
```

Comment:
why not just add a `getCounts` to FileSizeHistogram?

Reply:
I see ... we want it private ... and not touched or mutated ... this is fine I think